pkgs/tools/nix/nixpkgs-manual-mmdoc-watch/default.nix

```diff
@@ -0,0 +1,19 @@
+{ writeScriptBin, python3, entr } :
+let
+  np-build = writeScriptBin "np-build" ''
+    nix build .#nixpkgs-manual-mmdoc -L
+  '';
+in
+writeScriptBin "nixpkgs-manual-mmdoc-watch" ''
+  killbg() {
+  for p in "''${pids[@]}" ; do
+  kill "$p";
+  done
+  }
+  trap killbg EXIT
+  pids=()
+  ${python3}/bin/python -m http.server --directory ./result &
+  pids+=($!)
+  trap exit SIGINT
+  while true; do find doc pkgs/tools/nix/nixpkgs-manual-mmdoc/ | ${entr}/bin/entr -cd ${np-build}/bin/np-build; done
```

Comment:
Isn't entr Linux only and requires some setup? Last time I tried it, I couldn't get it to work.